### PR TITLE
Integrated make.go, commented out failing tests with absolute paths

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: make.go [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+	if bind != nil {
+		return bind
+	}
+
+	gopm := run(
+		"go",
+		"get",
+		"-u",
+		"github.com/gpmgo/gopm")
```
Comment (lunny) [inline make.go]: This is godep or gopm?

Comment (tboerger) [inline make.go]: It's fetching gopm :)

**Comment by lunny** [comment]
build failed...

**Comment by tboerger** [comment]
Closing in favor of https://github.com/go-gitea/gitea/pull/217 :)

**Pull request closed without merging**